app/com/linkedin/drelephant/spark/data/SparkRestDerivedData.scala

```diff
@@ -25,4 +25,5 @@ case class SparkRestDerivedData(
   stageDatas: Seq[StageData],
   executorSummaries: Seq[ExecutorSummary],
   stagesWithFailedTasks: Seq[StageData],
-  private[spark] val logDerivedData: Option[SparkLogDerivedData] = None)
+  private[spark] val logDerivedData: Option[SparkLogDerivedData] = None,
+  targetRestURI : String = "")
```

Comment:
From the usage of this property, name `targetRestURIForFailedTasks` or `targetRestURIForEF` seems more appropriate.

Reply:
I was thinking for making to general URI (instead for Failed Task or EF) thats why kept the name more abstract

Reply:
Yeah, storing General URI would be more intuitive than storing FailedTask URI (https://github.com/linkedin/dr-elephant/pull/640/files#r352433891)